docs/system/system-metrics.md

```diff
@@ -143,6 +144,34 @@ This metric is [recommended][MetricRecommended].
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+### Metric: `system.cpu.utilization`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.system.cpu.utilization -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability | Entity Associations |
+| -------- | --------------- | ----------- | -------------- | --------- | ------ |
+| `system.cpu.utilization` | Gauge | `1` | Fraction of total CPU time spent in active modes, aggregated across all system CPU cores. [1] | ![Development](https://img.shields.io/badge/-development-blue) | `host` |
+
+**[1]:** This metric provides a standardized and opinionated definition of CPU utilization.
+It is defined as the proportion of total system CPU time spent in non-idle and non-iowait modes — that is, the time spent in user, system, nice, interrupt, and steal modes.
+It is computed by aggregating `cpu.time` across all logical CPUs over a measurement interval,
```

Comment:
This calculation doesn't take into account the fact that all CPUs don't run at the same frequency: one core may be at 1GHz while another one is clocked at 2.5GHz at the same time.

A more precise calculation of a weighted average would be: `sum(delta(cpu.time{mode != "idle" and mode != "iowait"})) / sum(delta(cpu.time))`.

Reply:
But even the above calculation could be challenged, where one would want to see the utilization percentage based on the potential time when processor is running at full speed (or nominal speed, or whatever is meaningful).

If a core is running at half speed, it may show a 50% time utilization. But if it was running at full speed, it would have shown 25% time utilization for the same task. Knowing that clock speed varies based on the workload, maybe the 25% figure is more accurate than 50%? Who knows? 🤷‍♂️ 

My point being: I don't see the community agreeing on any definition of CPU utilization, as different operating systems already have very different definitions.

Reply:
@ChrsMark The definition doesn't specify it, and the implementation is in fact affected by processor speed... but doesn't know it. Again, what I'm saying is: let's just collect `system.cpu.time` and let the backend calculate processor utilization as a rate of this metric. It's more efficient and more accurate, and it lets end users decide what they want to see.